docs/reference/components/transforms/aggregate.cue

```diff
@@ -0,0 +1,188 @@
+package metadata
+
+components: transforms: aggregate: {
+	title: "Aggregate"
+
+	description: """
+		Aggregates multiple metric events into a single metric event based on a
+		the MetricKind. Incremental metrics are "added", Absolute uses last
+		value wins semantics.
```

Comment:
```suggestion
		Aggregates multiple metric events into a single metric event based on a defined interval window. This helps to reduce metric volume as the cost of granularity.
```

The description should be vague and the `how_it_works` section should deep dive into the details.